VisualRecognitionV4.playground/Pages/Analysis.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,48 @@
+//:## Analysis
+
+import Foundation
+import VisualRecognitionV4
+
+let visualRecognition = setupVisualRecognitionV4()
+
+//:### Analyze images
+
+let url = Bundle.main.url(forResource: "fruitbowl", withExtension: "jpg")
```

Comment:
- These files should be `honda.jpg` and `dice.png` to be consistent with other langs.
- Collection ID is `5826c5ec-6f86-44b1-ab2b-cca6c75f2fc7`
- no `threshold`

```curl
curl -X POST -u "apikey:{apikey}" -F "features=objects" -F "collection_ids=5826c5ec-6f86-44b1-ab2b-cca6c75f2fc7" -F "images_file=@honda.jpg" -F "images_file=@dice.png" "https://gateway.watsonplatform.net/visual-recognition/api/v4/analyze?version=2019-02-11"
```